styles/global.css

```
@@ -0,0 +1,103 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+a{
+  text-decoration: none;
+  color: inherit;
+}
+
+button {
+  background: none;
+  border: none;
+  outline: none;
+  box-sizing: none;
+  cursor: pointer;
+}
+
+img {
+  vertical-align: bottom;
+}
+
+
+body {
+  color: #374151;
+  word-break: keep-all;
+  font-family: "Pretenderd", sans-serif;
+}
+
+header {
+  width: 100%;
+  height: 70px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 200px;
+  background-color: #ffffff;
+  border-bottom: 1px solid;
```

P2:
디자인상에서는 회색 보더라인이니 아래와 같이 색상코드를 기제해주세요.

```suggestion
  border-bottom: 1px solid #DFDFDF;
```